### DISCUSSION TITLE
Plans for new release with fixes for 4285 & 4288

### DISCUSSION THREAD

**Discussion by sid-wr** [harfbuzz/harfbuzz#4606] [General] [answered]
Hi @behdad @khaledhosny 

Last release was in November.

Any plans for a new release with fixes for the following ?
- https://github.com/harfbuzz/harfbuzz/issues/4285
- https://github.com/harfbuzz/harfbuzz/issues/4288

**Comment by behdad** [comment]
Yeah I think it's a good time to cut a release. Donno how time passed...

**Reply by jschwe** [reply]
Any updates on a new release? I'd also be interested in a new release with https://github.com/harfbuzz/harfbuzz/pull/4585

**Reply by behdad** [reply]
Should be coming in the next day or two.

**Reply by khaledhosny** [reply, marked as answer]
https://github.com/harfbuzz/harfbuzz/releases/tag/8.3.1
